scripts/commitcheck.sh

```diff
@@ -68,6 +69,12 @@ function check_tagged_line_with_url()
 function new_change_commit()
 {
     error=0
+    type_subject=$(git log -n 1 --pretty=%s "$REF" | cut -d: -f1)
+	# shellcheck disable=SC2076
+	[[ " ${types[*]} " =~ " ${type_subject} " ]] && error=0 || error=1; \
+	echo "error: the commit subject line must start with a valid type, ${type_subject}
+	is not a recognized type. Valid types are: ${types[*}"
```

Comment:
Are you missing a `]` in `${types[*}`?

Reply:
Yes I am. 
Hard to spot, had to look 3 times to see it this time too... Thank you!